assemble/bin/accumulo-cluster

```diff
@@ -651,28 +639,46 @@ function main() {
       fi
       cat <<EOF >"$conf"/cluster.yaml
 manager:
-  - localhost
+  default:
+    conf_dir: /opt/accumulo/conf
+    extra_args: ""
+    hosts:
+      - localhost
 
 monitor:
-  - localhost
+  default:
+    conf_dir: /opt/accumulo/conf
+    extra_args: ""
+    hosts:
+      - localhost
 
 gc:
-  - localhost
+  default:
+    conf_dir: /opt/accumulo/conf
+    extra_args: ""
```

Comment:
Would be nice to document these options somehow in this file.  Is this only for accumulo sever process args?  Assuming it can not contain jvm args? If so could name it `server_args` which is a bit more specific.  Could possibly have a `jvm_args` in the future.

Reply:
Added documentation in 2e182d1

Reply:
> Could possibly have a `jvm_args` in the future.

Looking at `accumulo-env.sh`, a user can supply `ACCUMULO_JAVA_OPTS` environment variable to add arguments to the JVM. However, this is added before most of the arguments, not after. So, for example, setting a JVM Xmx size in the cluster.yaml file to override the accumulo-env.sh file, would not work. If the user wants to specify some arguments that are not in the accumulo-env.sh file at all, then that would likely work.